lib/std/os/bits/windows.zig

```diff
@@ -243,6 +243,19 @@ pub const IPPROTO_UDP = ws2_32.IPPROTO_UDP;
 pub const IPPROTO_ICMPV6 = ws2_32.IPPROTO_ICMPV6;
 pub const IPPROTO_RM = ws2_32.IPPROTO_RM;
 
+pub const pollfd = ws2_32.pollfd;
```

Comment:
Why duplicate these from ws2_32?

Reply:
So you can do `std.os.pollfd` etc and get the right thing. Also a lot of other `ws2_32` stuff is duplicated here, so I figured it'd be appropriate.